docs/howto/operate/ci-cd/index.md

```diff
@@ -19,3 +19,9 @@ aws.md
 ```{note}
 TODO: write specific documentation for google cloud and azure cloud providers
 ```
+
+## Setting up repo2docker-action
```

Comment:
We already have some info about using repo2docker action here https://github.com/2i2c-org/infrastructure/blob/master/docs/howto/customize/custom-image.md. Maybe we can have this "split up a repo" instructions there instead of creating a new topic in `hub operation`? 

Hmm, I'm not sure which one is better :( Personally I'd prefer to have the all info about repo2docker grouped, but at the same time, these splitting instructions are conceptually different than the info in `hub customization`. Hmm...